public/stylesheets/site/2.0/26-media-narrow.css

```diff
@@ -1,9 +1,8 @@
-/* MEDIA: only screen and (max-width: 640px), handheld ENDMEDIA (need ENDMEDIA because the media has more than one word)
-linearise content, suppress large images, suppress custom fonts and other large downloads */
+/* MEDIA: only screen and (max-width: 42em), handheld ENDMEDIA */
 
 #outer {
   background: #fff;
-  font-size: 14px;
+  font-size: 0.875em;
```

Comment:
Ems are preferred on mobile, too. 16px (the default font size in most browsers, even mobile) * 0.875 = 14px